CredentialsManagerTests/FilesTests.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using CredentialsManager;
+using FluentAssertions;
+using System.IO;
+using System.Linq;
+using CredentialsManager.FilesExceptions;
+using Xunit;
+
+namespace CredentialsManagerTests
+{
+  public class FilesTests
```

Comment:
Test names are very redundant. Consider adding two nested classes and per each method under test- makes test name less redundant. For example:
```cs
public class FilesTests
{
    public class ReadaAllText
    {
        Should_Throw_MyFileNotFoundException_When_Path_Is_Not_Present_Or_Not_Able_To_Open()
        {
          //...
        }
    }
}
```

Reply:
Shorter names and grouped by method by design ;)

Reply:
This is news to me, I actually did not know that you can nest classes 😲